html/pages/settings/webui.inc.php

```diff
@@ -31,6 +31,10 @@
           'descr'              => 'Use device groups filter',
           'type'               => 'checkbox',
     ),
+    array('name'               => 'webui.availability_map_box_size',
```

Comment:
This probably should just be moved to the widget settings if you can, allows each user to fully customise them.